networks/suzuka/suzuka-client/src/tests/indexer_stream.rs

```diff
@@ -0,0 +1,68 @@
+// pub mod alice_bob;
```

Comment:
Do we need to comment this out or just remove?

Reply:
It's part of a larger break out. For this PR, I can do whichever.